### PR TITLE
Make `noarch`

### LINKED ISSUE #1: Add `linux_ppc64le` packages
Opening to track addition of `linux_ppc64le` packages for `cuda`. This depends on `cuda-toolkit` packages of `linux_ppc64le` being added ( https://github.com/conda-forge/cuda-toolkit-feedstock/issues/1 ). So this is currently blocked atm

### LINKED ISSUE #4: Using `noarch`
As this package doesn't have any architecture dependency on its own, it would make sense to make this `noarch`. This would allow this package to be built once and reused on other architectures/OSes, which would simply the release process and cutdown on resource usage. For dependencies that are architecture/OS dependent, the right package would be pulled in. Raising to track evaluating this option

### PR DESCRIPTION
**Pull request by adibbley**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Fixes #4
Fixes #1

### CODE REVIEW

**Comment by adibbley** [comment]
@conda-forge-admin, please rerender

**Comment by adibbley** [comment]
@conda-forge-admin , please restart ci

File: conda-forge.yml [hunk 1]
```diff
@@ -4,5 +4,8 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+noarch_platforms:
+  - linux_64
+  - win_64
 provider:
   linux_aarch64: default
```
Comment (jakirkham) [inline conda-forge.yml:11]: Let's add `linux_ppc64le` here to ensure that platform is represented

```suggestion
provider:
  linux_aarch64: default
  linux_ppc64le: default
```

**Review by jakirkham — COMMENTED** [review]
Thanks Alex! 🙏 

Had one suggestion below

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender

**Review by jakirkham — APPROVED** [review]

**Comment by leofang** [comment]
Q: shouldn't we bump the build num?

**Comment by jakirkham** [comment]
That's a fair point Leo. Went ahead and pushed a `build/number` bump to `main` and cross-linked the PR in the commit message. Repeated this for the other `noarch` PRs

That said, since we are publishing a different architecture than the ones we already published, the packages will still upload. Users will still be able to download and install them (especially for the previously unsupported platforms)

Though Conda does sometimes issue warnings when downgrading or keeping the same `build/number`. So it is better to avoid the warning, which the new packages should do

**Pull request merged**